.github/ISSUE_TEMPLATE/-feature-request----title.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "[Feature Request] - Title"
+about: Suggest an idea for this project
+title: ''
+labels: Feature Request
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
Don't use BOLD statements.

headings should use heading markers, preferably H2

So replace ** with "##" at the beginning and remove the ** at the end of the line

Reply:
For our knowledge, is this primarily for linkability (i.e. I can go to an issue #is-your-feature-request-related-to-a-problem...) or for other reasons? Is there a docs/markdown guide/set of rules that you've followed in the past that this suggestion is from (i.e. are there other good suggestions we can adopt holistically?)